Makefile

```diff
@@ -8,6 +8,12 @@ SKOPEO_BRANCH = master
 # Set SUDO=sudo to run container integration tests using sudo.
 SUDO =
 
+GO ?= go
```

Comment:
So much of the `Makefile` uses `go` directly, do we need this variable? If we do, in which cases it should / shouldn’t be used?

Reply:
Valid point. Will drop it and use `go` for setting `GOBIN`.